tvm_block/src/accounts.rs

```diff
@@ -620,13 +626,15 @@ impl Account {
 
     pub fn active_by_init_code_hash(
         addr: MsgAddressInt,
+        dapp_id: UInt256,
```

Comment:
I suppose you could skip most of this constructor functions and just set default dapp id

Reply:
It would allow you not to change most of tests